microsite/src/main/resources/microsite/css/override.css

```diff
@@ -0,0 +1,26 @@
+.jumbotron {
+    background: url("../img/jumbotron_pattern.png") no-repeat fixed;
```

Comment:
also I hope this eventually goes away. The [new logo](https://github.com/scalaz/scalaz-zio/issues/150#issuecomment-445099306) is wonderfully intricate and subtle, it should take the centre stage, not some random picture that vaguely communicates ... something 🤔